package.json

```diff
@@ -92,14 +92,14 @@
     "eslint-plugin-no-only-tests": "3.3.0",
     "eslint-plugin-perfectionist": "4.6.0",
     "eslint-typegen": "1.0.0",
-    "h3": "npm:h3-nightly@2.0.0-1718872656.6765a6e",
+    "h3": "npm:h3-nightly@1.13.1-20250110-173418-de24917",
     "happy-dom": "16.6.0",
     "installed-check": "9.3.0",
     "jiti": "2.4.2",
     "knip": "5.42.2",
     "markdownlint-cli": "0.43.0",
     "memfs": "4.17.0",
-    "nitro": "npm:nitro-nightly@3.0.0-beta-28796231.359af68d",
+    "nitro": "npm:nitro-nightly@3.0.0-beta-28938837.19ec5395",
```

Comment:
v3 beta isn't ready for nuxt usage yet, I recommend avoiding bump for now and back to v2 for stability of nuxt nightly channel.

Reply:
this is the main branch

`nuxt-nightly@3x` is still tracking nitro v2.

Reply:
I'm aware.

Reply:
we've been tracking nitro-nightly since June 2024

https://github.com/nuxt/nuxt/pull/27702

Reply:
wondering are there any specific backward (forward is fine) differences? if yes, then yeah probably makes sense to move revert.

Reply:
we can leave things pinned at the previous nightly version if you think that's better, but this might also help us identify regressions in nitro's main branch, or things we need to change in Nuxt to adapt to it.

my motive for bumping the version in this PR is to make it possible at some point to run ecosystem-ci tests against nuxt/nitro.

Reply:
> wondering are there any specific backward (forward is fine) differences?

you can see the differences in [the PR](https://github.com/nuxt/nuxt/pull/27702), but they include server support for app-config, moving error handling utilities into Nuxt, addressing some type issues.

I plan to revisit once nitro is in rc status, and maybe update or revert some of those changes as needed. (particularly regarding error handling given our recent conversation)

it might be that some of them might not be needed to be reverted to go back to nitro v2 however - I'm very up for having the discussion

Reply:
yes, the new option for error fallback will also land for 2.11 (it is one of the current blockers that hasn't been released yet).

Really not sure about the current users of the nuxt `main` branch but if there are active users on it it could be more stable path. 

Or otherwise feel free to merge this PR, changes are fine.

Reply:
I'll merge this PR but then look to see if we can revert back to v2 until you feel the v3 branch is ready for nightly testing. just let me know when we're good to go! 🙏

Reply:
Cool. Actually let's keep it and see if need to revert if issues happen, I likely push some more progress this week.